### PR TITLE
joss-paper

### LINKED ISSUE #210: JOSS paper
See https://joss.readthedocs.io/en/latest/submitting.html and also https://github.com/eX-Mech/pymech/pull/23

When we will get close to 0.9.0, we will have to write a JOSS paper.

With our serious work on 0.8.0 and 0.9.0, it should not be too long to get a JOSS paper submitted (and even approved):

> If you’ve already developed a fully featured research code, released it under an [OSI-approved license](https://opensource.org/licenses), and written good documentation and tests, then we expect that it should take perhaps an hour or two to prepare and submit your paper to JOSS. But please read these instructions carefully for a streamlined submission.

Taken from https://joss.readthedocs.io/en/latest/submitting.html

I guess it should consist in writing few files ([paper.md](https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography), refs.bib, etc.) in a directory docs/joss_paper.

The submission process seems to be very simple and friendly https://joss.theoj.org/papers/new

### PR DESCRIPTION
**Pull request by akhoubani**
Fixes #210

### CODE REVIEW

**Comment by paugier** [comment]
@akhoubani I added some TODOs in paper.md. could you please work on them?

Could you please also fix the documentation build and remove the FIXME which leads to a code smell?

**Comment by ashwinvis** [comment]
You can tell sphinx to exclude them

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-exclude_patterns

**Comment by paugier** [comment]
We should add this Github Action: https://github.com/marketplace/actions/open-journals-pdf-generator

File: docs/joss_paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+and engineering disciplines. Nek5000 is a CFD Fortran code based on spectral element
+methods. It has a proven track record in numerous applications. In this article, we
+present Snek5000, a Python package designed to streamline the management and
+visualization of fluid dynamics simulations based on Nek5000. The package builds upon
+the functionality of Nek5000 by providing a user-friendly interface for launching and
+restarting simulations, loading simulation data, and generating figures and movies. This
+paper introduces Snek5000, discusses its design principles, and highlights its impact on
+the scientific community.
+
+## Snek5000: features and capabilities
```
Comment (paugier) [inline docs/joss_paper/paper.md:53]: It seems that summary has to be very short. So `Snek5000: features and capabilities` should not be in summary. I guess we should have a dedicated session.

```
# Snek5000: features and capabilities
```

**Comment by ashwinvis** [comment]
I will do that. I will also add a state of the art section

**Comment by paugier** [comment]
@ashwinvis could you please have a look at the text of the article? We'd like to submit quite soon.

**Pull request merged**